lib/cc/yaml/nodes/glob.rb

```diff
@@ -2,9 +2,6 @@ module CC
   module Yaml
     module Nodes
       class Glob < Scalar
```

Comment:
I think it does this already? https://github.com/codeclimate/codeclimate-yaml/blob/abh_normalize_glob_endings/lib/cc/yaml/nodes/ratings.rb#L5